### PR TITLE
RATEST-91: Resurrect bookRequestAppointment test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-91)

Description -> Resurrect bookRequestApppointment test

### CODE REVIEW

**Comment by kdaud** [comment]
Waiting for review cc: @sherrif10

File: ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java:69]: Why removing getAppointmentStatus() here .

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java:69]: Changed the assertion logic to rather assert the presence of the text Scheduled since it the text can only appear in the appointment table when an appointment has been successfully scheduled. The recent assertion implementation was relatively achieving the same BUT could not achieve it because was incorrectly constructed

**Review by sherrif10 — APPROVED** [review]
However we can merge this, The implementation is straight , however Lets have final conclusion of why getAppointmentStatus() is not working, probably we can improve that methd

File: ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java:69]: `getAppointmentStatus()` is checking whether its true that the appointment has been scheduled and if yes the status column  against the appointment within the appointment table should have the word `Scheduled`. Unfortunately `getAppointmentStatus().equals("Scheduled"))` does not achieve the intended goal and makes the test fail.  

We can rather use `assertTrue(manageAppointmentsPage.containsText("Scheduled")) `since the text Scheduled can only appear on the in the appointment table when the appointment has been successfully scheduled.

**Comment by kdaud** [comment]
Waiting for final review to see  this test enter into master build plan. cc: @dkayiwa

**Comment by sherrif10** [comment]
LGTM cc @dkayiwa  ,Thanks @kdaud

**Pull request merged**
